scripts/texturev.lua

```diff
@@ -117,9 +117,8 @@ project ("texturev")
 			"GLESv2",
 		}
 
-	configuration { "asmjs" }
+	configuration { "wasm*" }
 		kind "ConsoleApp"
-		targetextension ".bc"
```

Comment:
Not sure what was the history of setting targetextension to .bc, but the default scheme of using .o for object files and .a for libraries should work well now in upstream wasm backend, and will give wasm object files for fast incremental builds like on other platforms.